README.md

```diff
@@ -1 +1,37 @@
-# Hello-World
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+Hello world
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+
```

Comment:
- [ ] (Optional) It would be nice to include a Setup section in your `README.md` file where you describe how to clone your project and run it locally. It should loo like this:

```
git clone (ssh/http link)
cd <Folder(name of repository)>
Open with Vscode if it is installed
```